packages/client/mutations/handlers/handleAuthenticationRedirect.ts

```diff
@@ -5,20 +5,24 @@ import SendClientSideEvent from '../SendClientSideEvent'
 
 interface OnNextMeetingId extends OnNextHistoryContext {
   meetingId?: string | null
+  defaultRedirectPath?: string
 }
 
 const handleAuthenticationRedirect: OnNextHandler<
   AcceptTeamInvitationMutationReply$data | undefined,
   OnNextMeetingId
-> = (acceptTeamInvitation, {meetingId: locallyRequestedMeetingId, history, atmosphere}) => {
+> = (
+  acceptTeamInvitation,
+  {meetingId: locallyRequestedMeetingId, history, atmosphere, defaultRedirectPath = '/meetings'}
+) => {
   SendClientSideEvent(atmosphere, 'User Login')
   const redirectTo = getValidRedirectParam()
   if (redirectTo) {
     history.push(redirectTo)
     return
   }
   if (!acceptTeamInvitation?.team) {
-    history.push('/meetings')
+    history.push(defaultRedirectPath)
```

Comment:
+1 I'd just call this `redirectPath` because the default path is `/meetings`, but `/team/:teamId` isn't the default